app/controllers/newflow/login_signup_controller.rb

```diff
@@ -326,16 +327,16 @@ def change_password
     end
 
     def exit_accounts
-      referrer_params = Addressable::URI.parse(request.referrer).query_values.to_h
+      referrer_params = extract_params(request.referrer)
 
-      if (return_to = referrer_params['r'])
-        if Host.trusted?(return_to)
-          redirect_to(return_to)
+      if (r = referrer_params[:r])
```

Comment:
I would suggest a better name than r.  what is r?  it's good to write code thinking about how would a new dev look at this and instantly know what r is and what's going on

Reply:
👍  I'm thinking of documenting this param in the https://github.com/openstax/accounts#special-parameters section of the README and/or adding a comment to the controller action `exit_accounts`.